### ISSUE TITLE
Cannot parse URLs that contain `@`

### ISSUE THREAD

**Issue by JakeSidSmith** [peerigon/parse-domain#13]
I've been using your lib quite successfully until I came across a link with an `@` in it.

More specifically, this is a users profile on `https://medium.com/`. These are formatted like: `https://medium.com/@username/`.

Is this something you could fix easily, or is this intended (I assume it's something to do with emails)?

**Comment by JakeSidSmith** [comment]
Should I simply remove everything after the initial slash & parse that?

**Comment by sfnewzgirl** [comment]
@JakeSidSmith did that work for you? I think I might be running into the same issue.

**Comment by JakeSidSmith** [comment]
@sfnewzgirl I ended up using `url-parse` to split the URL before running it through `parse-domain`. Luckily I was already using this for some other parts of my project.

I pass `'/'` as the first argument (pathname) to `url-parse` to get the root of the domain.
The second argument is the base URL.

```javascript
var urlInfo = parseUrl('/', 'https://medium.com/@username/');
```

This gives me the following:
```javascript
{
  slashes: true,
  protocol: 'http:',
  hash: '',
  query: '',
  pathname: '/',
  auth: '',
  host: 'medium.com',
  port: '',
  hostname: 'medium.com',
  password: '',
  username: '',
  origin: 'http://medium.com',
  href: 'http://medium.com/'
}
```

And then I parse the href into domainParse

```javascript
var domain = parseDomain(urlInfo.href);
```

Which gives me the expected:

```javascript
{
  tld: 'com',
  domain: 'medium',
  subdomain: ''
}
```

This may be a little heavyweight for your project if you don't need any more URL info though. You could quite easily manually split after the domain name and then call `domain-parse` with this.

**Comment by jhnns** [comment]
Thx for reporting this. A fix shipped with `1.1.0`.
